### PR TITLE
test(): add test helper for mocking amqp components

### PR DESCRIPTION
**Pull request by jBouyoud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,58 @@ Bridge messaging lib
...
+```typescript
+    // Your base amqp instance with thait compagnon init method
+    import { amqp, initMessagingForConsumers } from 'src/messaging';
+    // Test Amqp components, typically procuder to test your consumer, etc...
+    import { testMessagingConfiguration } from './testMessagingConfiguration';
+
+    ...
+    
+    // Init Amqp mocks
+    amqpMockHelper.mockAmqpForJest(amqp);
```
Comment (ybonnefond) [inline README.md]: mockAmqpForTest?

File: README.md [hunk 2]
```diff
@@ -6,3 +6,58 @@ Bridge messaging lib
...
+## Testing
+
+For testing facilities you can use module `bridge-node-messaging/test`.
+In that module you can find an helper `amqpMockHelper` function that mocks all amqp interfaces.
+
+This helper initialize amqp mocking with `amqpMockHelper.mockAmqpForTest`.
+Mocking is based on third library called : `bridge-amqplib-mocks`.
+
+Sample use case: 
+In your test setup file : like Jest `setupTestFrameworkScriptFile`
```
Comment (ybonnefond) [inline README.md]: Most our projects run jasmine + js, that would be nice to have both 🃏

File: README.md [hunk 3]
```diff
@@ -6,3 +6,58 @@ Bridge messaging lib
...
+```typescript
+    // Your base amqp instance with thait compagnon init method
+    import { amqp, initMessagingForConsumers } from 'src/messaging';
+    // Test Amqp components, typically procuder to test your consumer, etc...
+    import { testMessagingConfiguration } from './testMessagingConfiguration';
```
Comment (ybonnefond) [inline README.md]: you should put a sample config also to be complete

File: README.md (hunk 2 above)
Comment (jBouyoud) [reply README.md]: Not yet tested, so I've no idea... I'll add a word on that

File: README.md (hunk 3 above)
Comment (jBouyoud) [reply README.md]: Same configuration as using this library. We should add a documentation of how we use this library.

File: test/lib/amqpMockHelper.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const amqplibMocks = require('bridge-amqplib-mocks');
+
+class AmqpMockHelper {
+    /**
+     * Mock the given Amqp components
+     * @param amqp : bridge-node-messaging/Amqp
+     */
+    mockAmqpForTest(amqp) {
```
Comment (ybonnefond) [inline test/lib/amqpMockHelper.js]: actually `ForTest`is probably not necessary

**Pull request merged**